### DISCUSSION TITLE
MixBarChartWithMinHeight

### DISCUSSION THREAD

**Discussion by codeharbor172** [recharts/recharts#3095] [General]
MinPointSize in mixBar does not work correctly.

**Comment by codeharbor172** [comment]
![image](https://user-images.githubusercontent.com/62288939/207570241-28a2a6e0-c3c1-4ac2-9a46-e21d1bd10360.png)

**Comment by ckifer** [comment]
hi @vlad19930514 

Can you give a reproduction other than just stating that it doesn't work correctly?
